testsuites/basic/lsvmbus.py

```diff
@@ -47,42 +46,46 @@ class LsVmBus(TestSuite):
             - Synthetic network adapter
             - Synthetic SCSI Controller
             - Synthetic IDE Controller (gen1 only)
-        2. check channel counts of each netvsc and storvsc SCSI device.
-        3. check expected channel count of each netvsc is min (num of vcpu, 8).
-        4. check expected channel count of each storvsc SCSI device is min
-            (num of vcpu/4, 64).
-        from the output of lsvmbus -vv command.
+        2. Check that each netvsc and storvsc SCSI device have correct number of vmbus
+            channels created and associated.
+            2.1 Check expected channel count of each netvsc is min (num of vcpu, 8).
```

Comment:
There was some suggestion of pointing to kernel's file:line to prove/back this here, no?